src/ios/CDVWKWebViewEngine.m

```diff
@@ -432,7 +432,7 @@ - (void) webView: (WKWebView *) webView decidePolicyForNavigationAction: (WKNavi
             // https://issues.apache.org/jira/browse/CB-12497
             int navType = (int)navigationAction.navigationType;
             if (WKNavigationTypeOther == navigationAction.navigationType) {
-                navType = (int)UIWebViewNavigationTypeOther;
+                navType = 5; // UIWebViewNavigationTypeOther
             }
```

Comment:
I would recommend removing the entire `if` condition completely.

Reply:
Also, 
* For UIWebView, `UIWebViewNavigationTypeOther` has the value of  `5`
* For WKWebView, `WKNavigationTypeOther` has the value of `-1`

This change wouldn't have worked as expected.

Reply:
In fact, it can't be removed or it will stop working in older cordova-ios versions, can only be removed for cordova-ios 6